app/src/main/res/values-en/strings.xml

```diff
@@ -925,12 +925,12 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_kerb_height_lowered_ramp">Curb ramp</string>
     <string name="quest_kerb_height_no">No curb at all</string>
 
-    <string name="quest_lanes_title">"How many lanes for cars does this road have?"</string>
+    <string name="quest_lanes_title">"How many car lanes does this road have?"</string>
     <string name="quest_lanes_answer_lanes">"Marked lanes"</string>
-    <string name="quest_lanes_answer_lanes_odd2">"Lane count differs for each side"</string>
-    <string name="quest_lanes_answer_lanes_description2">"Specify the total number of marked lanes (both directions) usable by car traffic. So, bike and dedicated parking lanes don’t count.
+    <string name="quest_lanes_answer_lanes_odd2">"Lane count differs on each side"</string>
```

Comment:
This sounds more natural, but it’s more verbose. Your call:

```suggestion
    <string name="quest_lanes_answer_lanes_odd2">"Each side has a different number of lanes"</string>
```

Reply:
"different number of lanes" is far clearer IMO.